aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,179 @@
+            schedule_params = {
+                "interval": 3,
+                "timeUnit": "hour",
+                "startTime": int(time.time())
+            }
+        else:
+            schedule_params = {
+                "interval": 1,
+                "timeUnit": "day",
+                "startTime": int(time.time() + 60*60*24*365)
+            }
+        flow_spec_id = "cd2fc47e-e838-4f38-a581-8fff2f99b63a"
+        flow_obj = {
+            "name": f"[CMLE][Week2] Flow for Featurized Dataset to DLZ created by {self.username}",
+            "flowSpec": {
+                "id": flow_spec_id,
+                "version": "1.0"
+            },
+            "sourceConnectionIds": [
+                source_connection_id
+            ],
+            "targetConnectionIds": [
+                target_connection_id
+            ],
+            "transformations": [],
+            "scheduleParams": schedule_params
+        }
+        flow_res = self.flow_conn.createFlow(
+            obj = flow_obj,
+            flow_spec_id = flow_spec_id
+        )
+        try:
+            dataflow_id = flow_res["id"]
+            print("dataflowId " + dataflow_id + " has been created")
+        except KeyError:
+            raise RuntimeError(f"Error when creating data flow: {flow_res}")
+
+        #save dataflow_id to config
+        self.utils.save_field_in_config("Platform", "dataflow_id", dataflow_id)
+        return dataflow_id
+
+    def createFlowRun(self, dataflow_id, dataset_id, on_schedule):
+        if not on_schedule:
+            res = self.retryOnNotReadyException(dataflow_id = dataflow_id, dataset_id= dataset_id)
+            flowRunId = res["destinations"][0]["datasets"][0]["statusURL"].rsplit('/', 1)[-1]
+            #check if flowRun has finished
+            finished = False
+            while not finished:
+                try:
+                    run = self.flow_conn.getRun(runId = flowRunId)["items"][0]
+                    run_id = run["id"]
+                    run_started_at = run["metrics"]["durationSummary"]["startedAtUTC"]
+                    run_ended_at = run["metrics"]["durationSummary"]["completedAtUTC"]
+                    run_duration_secs = (run_ended_at - run_started_at) / 1000
+                    run_size_mb = run["metrics"]["sizeSummary"]["outputBytes"] / 1024. / 1024.
+                    run_num_rows = run["metrics"]["recordSummary"]["outputRecordCount"]
+                    run_num_files = run["metrics"]["fileSummary"]["outputFileCount"]
+                    print(f"Run ID {run_id} completed with: duration={run_duration_secs} secs; size={run_size_mb} MB; num_rows={run_num_rows}; num_files={run_num_files}")
+                    finished = True
+                except Exception as e:
+                    print(f"No runs completed yet for flow {self.dataflow_Id}")
+                    time.sleep(30)
+            #push the startDate to a year from now
+            startTime = int(time.time() + 60*60*24*365)
```

Comment:
@cmenguy I have checked with Anton offline regarding the appropriate date to put, he suggested us to be conservative about the value here since there is a plan to put validation around the start date to be within a reasonable range. 100 years might be a invalid input once validation is in place while 1 year is safer.